roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -149,12 +149,33 @@
     with_items:
       - {src: "{{ he_local_vm_dir }}/vm.conf", dest: /var/run/ovirt-hosted-engine-ha}
       - {src: "{{ he_local_vm_dir }}/hosted-engine.conf", dest: /etc/ovirt-hosted-engine/}
+  - name: Verify fapolicyd.rules file
+    stat:
+      path: /etc/fapolicyd/fapolicyd.rules
+    register: fapolicy_rules
+  - name: Add rule to fapolicyd.rules file
+    blockinfile:
+      path: /etc/fapolicyd/fapolicyd.rules
+      block: |
+        # temporary entry added by hosted-engine-setup for running commands as vdsm user
+        allow perm=any uid=36 : dir=/var/tmp/
+      insertbefore: "^# Allow any program to open trusted language files"
+      marker: "## {mark} HOSTED-ENGINE-SETUP MANAGED BLOCK ##"
+      backup: yes
+    when: fapolicy_rules.stat.exists
+  - name: Restart fapolicyd service
+    service:
+      name: fapolicyd
+      state: restarted
+    when: fapolicy_rules.stat.exists
```

Comment:
This checks if the config exists, but the service might not be running it might be just installed.

Reply:
Thanks, I'll also check if the service is running.